docs/getting_started.html

```diff
@@ -0,0 +1,212 @@
+  <span class="na">name</span><span class="pi">:</span> <span class="s">ssh</span>
+
+<span class="na">verifier</span><span class="pi">:</span>
+  <span class="na">name</span><span class="pi">:</span> <span class="s">terraform</span>
+  <span class="na">groups</span><span class="pi">:</span>
+    <span class="pi">-</span> <span class="na">name</span><span class="pi">:</span> <span class="s">example</span>
+      <span class="na">controls</span><span class="pi">:</span>
+        <span class="pi">-</span> <span class="s">example</span>
+
+<span class="na">platforms</span><span class="pi">:</span>
+  <span class="pi">-</span> <span class="na">name</span><span class="pi">:</span> <span class="s">terraform</span>
+
+<span class="na">suites</span><span class="pi">:</span>
+  <span class="pi">-</span> <span class="na">name</span><span class="pi">:</span> <span class="s">example</span>
+</code></pre></div>        </div>
+        <div class="tab-pane fade" id="list-three" role="tabpanel" aria-labelledby="list-three-list">
+          Create this file <p class="font-weight-bold" style="color: #32c850; display: inline;">main.tf</p> and add the block of code into it.
+          <br><br>
+<div class="highlight"><pre class="syntax-highlight ruby"><code><span class="n">resource</span> <span class="s2">"null_resource"</span> <span class="s2">"example"</span> <span class="p">{</span>
+  <span class="n">provisioner</span> <span class="s2">"local-exec"</span> <span class="p">{</span>
+    <span class="n">command</span> <span class="o">=</span> <span class="s2">"echo 'this is my first test' &gt; foobar"</span>
+  <span class="p">}</span>
+<span class="p">}</span>
+</code></pre></div>          Create Terraform fixture code that calls the module's Terraform code to help simulate the module's code.
+          <br><br>
+          Create this file <p class="font-weight-bold" style="color: #32c850; display: inline;">test/fixtures/example/main.tf</p> and add the block of code into it.
+          <br><br>
+<div class="highlight"><pre class="syntax-highlight ruby"><code><span class="n">module</span> <span class="s2">"example"</span> <span class="p">{</span>
+  <span class="n">source</span> <span class="o">=</span> <span class="s2">"../../.."</span>
+<span class="p">}</span>
+</code></pre></div>          Run the Terraform module code with kitchen converge
+          <br><br>
+<div class="highlight"><pre class="syntax-highlight ruby"><code><span class="n">bundle</span> <span class="nb">exec</span> <span class="n">kitchen</span> <span class="n">converge</span>
+</code></pre></div>        </div>
+        <div class="tab-pane fade" id="list-four" role="tabpanel" aria-labelledby="list-four-list">
+          We've created the Terraform code, now it's time to create the Inspec control tests. Please see the <a href="https://www.inspec.io/docs/reference/profiles/" style="color: #32c850;">Inspec documentation</a> to learn more about profiles and controls.
```

Comment:
The rest of the guide avoids use of conversational words like "we've" and I think we should stick to that.